constraint/pkg/client/client.go

```diff
@@ -619,10 +623,27 @@ func (c *Client) RemoveData(ctx context.Context, data interface{}) (*types.Respo
 	return resp, &errMap
 }
 
-// Review makes sure the provided object satisfies all stored constraints.
+// Review makes sure the provided object satisfies constraints applicable for specific enforcement points.
 // On error, the responses return value will still be populated so that
 // partial results can be analyzed.
-func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.QueryOpt) (*types.Responses, error) {
+func (c *Client) Review(ctx context.Context, obj interface{}, opts ...reviews.ReviewOpt) (*types.Responses, error) {
+	var eps []string
+	cfg := &reviews.ReviewCfg{}
+	for _, opt := range opts {
+		opt(cfg)
+	}
+	if cfg.EnforcementPoint == "" {
```

Comment:
This is the enforcementPoint under scopedEnforcementActions right?

Reply:
This is enforcement point passed in `review.opts`.